### DISCUSSION TITLE
Guacamole?

### DISCUSSION THREAD

**Discussion by jeeftor** [tteck/Proxmox#2707] [Ideas]
Any chance we could get a Guacamole container built out? I think this could be useful

**Comment by NorthantsAndy** [comment]
Seconded!

https://guacamole.apache.org/doc/gug/guacamole-docker.html

**Comment by Narkowii** [comment]
Guacamole in LXC with this video, works fine for me !

https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.youtube.com/watch%3Fv%3DcugExjVmQT4&ved=2ahUKEwj-3YmJ4JmFAxWNRKQEHaxjDhoQwqsBegQIDxAG&usg=AOvVaw0sGuqJh4jhcstKGd293AVx
